trainings/tasks.py

```diff
@@ -0,0 +1,23 @@
+from celery.task import Task
+from celery.utils.log import get_task_logger
+
+
+logger = get_task_logger(__name__)
+
+
+class The_Incr(Task):
```

Comment:
Hmm, is part of cookie cutter, I'll just leave it since I think I'll use it soon. Will add a note to check if it's redundant at the end.